how-we-process-data-five-times-more-efficiently-using-a-scale-out-mysql-alternative.md

```diff
@@ -0,0 +1,138 @@
+---
+title: 'How We Process Data Five Times More Efficiently Using a Scale-Out MySQL Alternative'
+author: ['Yu Liu']
+date: 2020-11-19
+summary: A senior architect at China Telecom Bestpay shared how TiDB, a MySQL-compatible, distributed SQL database, boosted the performance of their application system by 3–5 times.
+tags: ['MySQL compatibility', 'Scalability']
+url: /success-stories/how-we-process-data-five-times-more-efficiently-using-a-scale-out-mysql-alternative/
+customer: China Telecom
+customerCategory: Financial Services
+categories: ['MySQL Scalability']
+logo: /images/blog/customers/china-telecom-bestpay-logo.png
+image: /images/blog/bestpay-tidb-case-study-banner.jpg
+---
+
+**Industry:** Mobile Payment
+
+**Author:** Yu Liu (Senior Architect, China Telecom Bestpay)
+
+![How We Process Data Five Times More Efficiently Using a Scale-Out MySQL Alternative](media/bestpay-tidb-case-study-banner.jpg)
```

Comment:
Next time, you can make your ALT text shorter here, for example, "a scale-out MySQL alternative," or "MySQL horizontal scaling" and use an image name that contains an SEO keyword, like "bestpay-mysql-horizontal-scalability."

Reply:
This is a banner image that contains no information at all. I think It makes no sense to add the SEO keyword to its filename? On the SEO side, I think the markdown filename already makes the effect.

Reply:
The ALT text of an image also matters to SEO.

Reply:
I mean the image name, not the ALT text. The ALT text looks fine to me. My point here is that the primary principle we should follow is that the image name should describe the image.

Reply:
Yes. We should use a descriptive image name. If it contains an SEO keyword, it will be better. This is an SEO strategy.

Reply:
Okay, then I will use `bestpay-tidb-case-study-banner.jpg` this time because an SEO keyword is not suitable for describing a banner image. Next time for informative images I'll embed the SEO strategy.